app/scripts/extensions/javalink/javaLink.js

```diff
@@ -66,7 +66,9 @@
             var title = container.name || 'Untitled Container';
             var token = AuthService.UserStore().getToken() || '';
             var targetURI = new URI().path(BaseHref)
-                                     .segment('java/') // Must have a trailing slash to avoid runtime errors in Safari
+                                     .segment('java')
+                                     // Add an empty segment to force a trailing slash, which is required.
+                                     .segment('/')
```

Comment:
I believe this should be

```js
.segment('')
```

See https://github.com/medialize/URI.js/commit/2b604927f997299789c50badadbbb295c3bab754